src/transformers/models/seaformer/feature_extraction_seaformer.py

```diff
@@ -0,0 +1,33 @@
+# coding=utf-8
```

Comment:
Please see my previous comments, we are deprecating the FeatureExtractor classes for vision models, you can delete this file